lib/std/leb128.zig

```diff
@@ -3,7 +3,7 @@ const testing = std.testing;
 
```

Comment:
Consider a deprecation period for a smoother transition, by adding these for each function:
```zig
/// Deprecated: use `readUleb128`
pub const readULEB128 = readUleb128;
```

Allows projects to move to the new API sooner by not forcing them to break compatibility with the current version.

Reply:
Done. What's the general guidance on when to remove this? Just curious.